### DISCUSSION TITLE
Provide / Inject theme not working ?

### DISCUSSION THREAD

**Discussion by Drozerah** [ecomfe/vue-echarts#722] [Q&A]
Hi! using the Vue-ECharts provide/inject API and vue 3 composition API, I'not able to set the chart theme, yet I'am following https://stackblitz.com/edit/vue-echarts-vue-3?file=src%2FApp.vue sandbox at line 25, replacing 

`provide(THEME_KEY, 'dark');` by  `provide(THEME_KEY, 'vintage');`  and I have not result according to the 'vintage' theme provided by Echarts itself, can anyone help me or confirm ?

**Comment by Euraxluo** [comment]
我导入之后,tailwindcss都失效了
你知道是什么原因吗

**Comment by kingyue737** [comment]
Only the default theme, `light` and `dark` are built-in.
Check out the docs: https://echarts.apache.org/en/tutorial.html#Overview%20of%20Style%20Customization

`vintage` theme is not provided by Echarts itself. You need to include it manually in your code, for example:
```ts
import { registerTheme } from 'echarts/core'
import themeJSON from '/xxx/xxx/vintage.json'

registerTheme('vintage', themeJSON)
```
